sensor_msgs/Image nodeA
geometry_msgs/Point TBtoA
std_msgs/Int32 pixel_x
std_msgs/Int32 pixel_y
---
std_msgs/Int32 response